App/Reducers/ReducersET.ts

```diff
@@ -0,0 +1,89 @@
+import { Reducer, combineReducers } from 'redux';
+import { ACTION, FollowStateTypes, Stage } from './ActionsET';
+ /**
+  * Represents the part of the state for the new WorkItem
+  * @interface IWorkItem
+  */
+export interface IWorkItem {
```

Comment:
I am going to use the account, project, and area from Miranda's part of the store.